data/core/help.cfg

```diff
@@ -390,7 +390,58 @@ If a strike is determined to hit, it will always do at least 1 point of damage.
+            _ "From left to right:" + "
+" + _ "• The seven-pointed star shape denotes a unit that can recruit." + "
+" + _ "• Most units have a circular shape." + "
+" + _ "• Unit without a <ref>dst='movement' text='zone of control'</ref> have a broken version of the shape." + "
+" + _ "• Some campaigns start with a level zero leader. Able to recruit but lacking a ZoC, these units have a broken version of the seven-pointed star." + "
+" + _ "• Hero units appear in campaigns with a softer variant of the seven-pointed star."
+    [/topic]
+    # wmllint: markcheck on
+
+    # wmllint: markcheck off
+    [topic]
+        id=crowns
+        title= _ "Crowns and loyal markers"
+        text=_ "Some units will have an additional marker superimposed on their <ref>dst='orbs' text='orb'</ref>, and a corresponding shape to their <ref>dst='orbs' text='ellipses'</ref>:" + "
+
+" +
+            "<img>src='help/empty-square.png~CROP(0,0,72,36)~BLIT(misc/leader-crown.png~CROP(4,0,36,18)~XBRZ(2))'</img>" +
+            "<img>src='help/empty-square.png~CROP(0,0,72,36)~BLIT(misc/hero-icon.png~CROP(4,0,36,18)~XBRZ(2))'</img>" +
+            "<img>src='help/empty-square.png~CROP(0,0,72,36)~BLIT(misc/loyal-icon.png~CROP(4,0,36,18)~XBRZ(2))'</img>" +
+            "<img>src='help/empty-square.png~CROP(0,0,72,36)~BLIT(misc/leader-expendable.png~CROP(4,0,36,18)~XBRZ(2))'</img>" + "
+" +
+            "<img>src='help/empty-square.png~BLIT(misc/ellipse-leader-top.png)~BLIT(misc/ellipse-leader-bottom.png)~BLIT(units/orcs/leader.png~TC(1, magenta))~BLIT(misc/leader-crown.png)'</img>" +
+            "<img>src='help/empty-square.png~BLIT(misc/ellipse-hero-top.png)~BLIT(misc/ellipse-hero-bottom.png)~BLIT(units/dunefolk/herbalist/alchemist.png~TC(1, magenta))~BLIT(misc/hero-icon.png)'</img>" +
+            "<img>src='help/empty-square.png~BLIT(misc/ellipse-top.png)~BLIT(misc/ellipse-bottom.png)~BLIT(units/human-loyalists/spearman.png~TC(1, magenta))~BLIT(misc/loyal-icon.png)'</img>" +
+            "<img>src='help/empty-square.png~BLIT(misc/ellipse-leader-top.png)~BLIT(misc/ellipse-leader-bottom.png)~BLIT(units/human-magi/white-mage.png~TC(1, magenta))~BLIT(misc/leader-expendable.png)'</img>" + "
+
+" + _ "• Leaders are shown with a <bold>text='gold crown'</bold>" + "
+" + _ "• Heroes have a <bold>text='blueish-silver crown'</bold>. Losing any of these units is likely to result in an immediate defeat." + "
+" + _ "• <ref>dst='traits_loyal' text='Loyal units'</ref> have the <bold>text='loyal icon'</bold>. Most campaigns will give you a few loyals, however they never occur randomly — they are always characters that are a part of the narration, never random recruits. Losing them doesn’t result in automatic defeat." + "
+" + _ "• Expendable leaders are shown with a <bold>text='silver crown'</bold>. Rarely encountered in campaigns, these units can recruit but losing them doesn’t automatically lead to a defeat." + "
```

Comment:
Half this text needs to be removed - I copied it to a separate paragraph, but forgot to trim the original paragraph.